src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/resample/resample_kernel_opt.cpp

```diff
@@ -116,17 +139,19 @@ bool ResampleKernelOpt::Validate(const Params& p, const optional_params& o) cons
         return false;
 
     const auto& input = params.inputs[0];
+    const auto & output = params.outputs[0];
 
     if ((input.GetDType() == Datatype::UINT8 || input.GetDType() == Datatype::INT8) &&
         params.resampleType != ResampleType::NEAREST_NEIGHBOR &&
-        params.resampleType != ResampleType::BILINEAR_INTERP)
+        params.resampleType != ResampleType::BILINEAR_INTERP &&
+        params.resampleType != ResampleType::LINEAR_ONNX)
         return false;
 
-    if (input.GetLayout() != DataLayout::fs_b_yx_fsv32 &&
-        input.GetLayout() != DataLayout::b_fs_yx_fsv16 &&
-        input.GetLayout() != DataLayout::b_fs_yx_fsv32 &&
-        input.GetLayout() != DataLayout::bs_fs_yx_bsv32_fsv16 &&
-        input.GetLayout() != DataLayout::bs_fs_yx_bsv32_fsv32)
```

Comment:
Does it mean the size of axis z should be same between input and output when input is 5D? According to the specification, when it is a 5D tensor, it is possible to resample the x, y, and z axes.

Reply:
We got a list of shapes that must be optimized for linear_onnx, so we added support only for resampling X and Y axes.
Resampling Z axes can be added to the existing resampl_opt kernel but will decrease the existing performance for requested shapes.

During today's status meeting, we agreed to add a resampling Z axe as separate PR, for this purpose will be created new CPP and cl kernel files.